contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,101 @@
+
+### The problem? Communication is messy
+
+Making remote work actually work requires making big cultural and procedural changes. Fixing how your company communicates is one of the hardest as it requires everyone to buy in and change their behavior. That means saying goodbye to email, setting fire to Microsoft Teams, and training people to be much more effective communicators. Here’s how.
+
+### The solution: Write. Everything. Down
+
+How do I [file expenses](https://posthog.com/handbook/people/spending-money)? What’s the [onboarding process for new hires](https://posthog.com/handbook/people/onboarding)? What’s the product analytics team [focused on this quarter](https://posthog.com/teams/product-analytics)? In most companies, answers to these kinds of questions are either hidden in labyrinthian collections of out-of-date Google Docs or inside someone’s brain. At well-functioning, all-remote companies, they’re written down in a handbook where anyone can access them. That handbook can be your public website, or an internal wiki. But written communication also includes Slack, GitHub, or systems like Trello or Airtable.
+
+Defaulting to written communication means, chances are, if someone has a question they don’t have to wait for an answer – they can go find it themselves. And if they can’t find an answer, writing a clear, concise question in a public channel on Slack will ensure that everyone can see the answer. Being able to clearly communicate often complex ideas is a key skill for anyone working remotely, but especially engineers. Software engineer Dmitry Kudryavtsev [sums it up nicely](https://www.yieldcode.blog/post/why-engineers-should-write/): “The sooner you get better at writing, the faster you will become a better engineer.” It also created some lively discussion on [Hacker News](https://news.ycombinator.com/item?id=36659166).
+
+Automattic’s Beau Lebens has written (of course) a [great guide](https://beau.blog/2020/03/remote-work-at-scale/) to the benefits of effective written communication with lots of practical tips. One key takeaway: tools are not the answer. Installing a bunch of bots to transcribe meetings and run automated functions in Slack will only work if you do all the hard work first. And nailing how you communicate is a big part of that. Writing things down is great because:
```

Comment:
As per earlier comment, would prefer to have more than one takeaway here, probably bullet point them. Can replace / combine if the following section if that makes sense.

Reply:
Sorry, don't quite follow. Do you mean replacing/reworking the bullets on written communication with bullets that cover both written communication and other points/takeaways? If so, what do you reckon those could be?